gitcoins-cordapp/contracts/src/main/kotlin/com/gitcoins/contracts/GitTokenContract.kt

```diff
@@ -0,0 +1,16 @@
+package com.gitcoins.contracts
```

Comment:
You really don't need this file :D

Reply:
Looks like they must've fixed the bug because you're right!